content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -17,29 +17,119 @@ example application for this task.
 
 ## Before you begin
 
+Istio provides a choice of backend tracing providers.
+
+<details><summary>Jaeger or Zipkin</summary>
```

Comment:
Do not use these HTML tags. They are not uses anywhere else in the docs and they just introduce inconsistency.